contracts/SpokePool.sol

```diff
@@ -1039,16 +1237,17 @@ abstract contract SpokePool is
         SpokePoolInterface.RelayerRefundLeaf memory relayerRefundLeaf,
         bytes32[] memory proof
     ) public payable virtual override nonReentrant {
-        _preExecuteLeafHook(relayerRefundLeaf.l2TokenAddress);
+        _preExecuteLeafHook(relayerRefundLeaf.l2TokenAddress.toBytes32());
```

Comment:
I kept it to prevent extra castings in the [_distributeRelayerFunds](https://github.com/across-protocol/contracts/blob/48e402374969662ea666192a7926d7f8c8b08949/contracts/SpokePool.sol#L1366) function but you are right, I think these are negligible so I've updated the interface now.